coulttxml

```diff
@@ -149,25 +154,79 @@ def main():
         print("Error: The number of different residues in the XML does not match the count from the PDB or PDBx file.")
         sys.exit(1)
 
-    with open('addCoulTT.py', 'w') as f:
-        f.write(before)
-        # Loop over each residue and its corresponding count from the PDB
-        f.write("# Put flag 1 in 1st or 2nd field in atoms to be screened\n")
-        f.write("# donor (1st field) is the charge, acceptor (2nd field) the induced dipole\n")
-        for residue_xml, count in zip(residue_names_xml, num_molecules):
-            f.write(f"for i in range({count}):\n")
-            atoms = next((res for res in residues_xml if res.attrib['name'] == residue_xml), None).findall('Atom')
-            for i, atom in enumerate(atoms):
-                name = atom.attrib['name']
-                charge = float(atom.attrib['charge'])
-                if not args.core and i < len(atoms) - 1:
-                    nxtname = atoms[i+1].attrib['name']
-                    if nxtname.startswith('D'):
-                        charge = - float(atoms[i+1].attrib['charge'])
-                f.write(f"    CoulTT.addParticle([0, 0, {charge:8.5f}]) # {name:4s} {atom.attrib['type']:12s}\n")
-        f.write(after)
-
-    print("OpenMM code for CoulTT written to addCoulTT.py")
+    if args.s is not None:
+        with open("omm-p-tt.py", 'w') as outputfile:
+            with open(args.s, 'r') as file:
```

Comment:
Reading the omm-p.py script into data can be before and outside the 'with open("omm-p-tt.py")' structure, to avoid excessive indent